nixos/modules/services/monitoring/librenms.nix

```
@@ -354,7 +376,11 @@ in
       description = ''
         Attrset of the LibreNMS configuration.
         See <https://docs.librenms.org/Support/Configuration/> for reference.
-        All possible options are listed [here](https://github.com/librenms/librenms/blob/master/misc/config_definitions.json).
+
+        All possible options and their defaults are listed [here](https://github.com/librenms/librenms/blob/master/misc/config_definitions.json).
```

```suggestion
        All possible options and their defaults are listed [here](https://github.com/librenms/librenms/blob/master/resources/definitions/config_definitions.json).
```
It seems like the file was moved.

---

Ah right, that's why I used the absolute links in the documentation.
I will create an additional commit for this.